lumicks/pylake/channel.py

```diff
@@ -504,6 +505,123 @@ def plot(self, start=None, **kwargs):
         plt.ylabel(self.labels.get("y", "y"))
         plt.title(self.labels.get("title", "title"))
 
+    def highlight_time_range(
+        self,
+        item,
+        color=None,
+        annotation=None,
+        annotation_direction=None,
+        label=None,
+        start=None,
+        alpha=0.15,
+        font_color=None,
+    ):
```

Comment:
let's make these keyword only to avoid same headaches we created in the past
```suggestion
    def highlight_time_range(
        self,
        item,
        *,
        color=None,
        annotation=None,
        annotation_direction=None,
        label=None,
        start=None,
        alpha=0.15,
        font_color=None,
    ):
```